shared/utils.py

```diff
@@ -102,7 +102,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, *a, **k):
-        self.fd.seek(0, 3)          # go to end
+        self.fd.seek(0, 2)          # go to end
```

Comment:
this would have been a bug; is there a test case that triggers/demos the bug before/after fix?

Reply:
- looks like `seek(0,3)` was a noop based on my reading of `external/micropython/extmod/vfs_fat_file.c:112`
- so change looks like good fix of a bug that never had impact